java/code/src/com/redhat/rhn/domain/user/UserFactory.java

```diff
@@ -708,7 +708,7 @@ public void setUserServerPreferenceValue(User user,
      */
     public List<User> findAllUsers(Optional<Org> inOrg) {
         Map<String, Object> params = new HashMap<>();
-        int orgId = Opt.fold(inOrg, () -> -1, org -> org.getId().intValue());
+        int orgId = Opt.fold(inOrg, () -> null, org -> org.getId().intValue());
```

Comment:
`int` cannot be null. So maybe we need to work with `Integer` here?

Reply:
Fixed, I missed that. Now the problem is:
``` 
[junit] 	at org.hibernate.query.internal.AbstractProducedQuery.list(AbstractProducedQuery.java:1398)
[junit] Caused by: org.postgresql.util.PSQLException: ERROR: operator does not exist: numeric = bytea
[junit]   Hint: No operator matches the given name and argument type(s). You might need to add explicit type casts.
```
Not sure if this is something with Postgres, or with the query hibernate generates